### ISSUE TITLE
Hyperparameter Tuning

### ISSUE THREAD

**Issue by nikhilravikrishnan** [nikhilravikrishnan/AudioCaptioning#4]
- Normalization
- Pre-training
